src/tests/actions/socialLoginAction.test.js

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import moxios from 'moxios';
+import configureMockStore from 'redux-mock-store';
+import thunk from "redux-thunk";
+import AxiosConfig from '../../AxiosConfig';
+import socialLoginAction from '../../store/actions/authActions/socialLoginAction';
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+const store = mockStore();
+
+const mockSuccessResponse = {
+    status: 200,
+    data: {
+        user: { token: "thisisatokenforyou"}
+
+    }
+};
+
+describe("network calls", () => {
```

Comment:
@cob04, make this more descriptive, use something like `OAuth network call` instead of `network calls`.